clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/ApiClientBuilder.java

```diff
@@ -97,6 +97,8 @@ public ApiClientBuilder(URI apiEndPoint, Class<T> clientClass, ExceptionBuilder
 
     private Integer maxRequestsPerHost = 5;
 
+    private boolean usingDefaultLogger = true; // Just kept the same behavior as the original code
```

Comment:
This comment is redundant, people who do not see this PR will be confused by the term original code and should be deleted.

Reply:
i deleted it already